File: python/7 kyu/218.py
# https://www.codewars.com/kata/558ee8415872565824000007/train/python

# Passed

def is_divisible(n, *other_numbers):    
    for num in other_numbers:
        if n % num != 0:
            return False
    return True

output = is_divisible(12, 3, 4)
print(output)

File: nodejs/8 kyu/218.js
// https://www.codewars.com/kata/57a2013acf1fa5bfc4000921/train/javascript

// Passed

function findAverage(array) {
  let avg;
  if (array.length === 0) {
    avg = 0;
  } else {
    let sum = array.reduce((prev, curr) => prev + curr, 0);
    avg = sum / array.length;
  }

  return avg;
}

const output = findAverage([1, 2, 3, 4]);
console.log(output);